sdk/storage/storage-blob/tests-resources.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "uniqueId": {
+      "type": "String"
+    },
+    "endpointSuffix": {
+      "type": "string",
+      "defaultValue": "core.windows.net"
+    }
+  },
+  "variables": {
+    "storageApiVersion": "[providers('Microsoft.Storage', 'storageAccounts').apiVersions[0]]",
```

Comment:
After further reading we're opting to move away from computing the API version. We should use a properly specified version.

In places where there are multiple of the same resources we should use a constant variable (like the latest code here: https://github.com/Azure/azure-sdk-for-java/blob/feature/live-test-resource-mgmt/sdk/storage/test-resources.json#L14)... In this case just delete `storageApiVersion`